vm/mx.vm/mx_vm.py

```diff
@@ -962,6 +962,9 @@ def contents(self):
             if any((' ' in arg for arg in build_args)):
                 mx.abort("Unsupported space in launcher build argument: {} in config for {}".format(image_config.build_args, image_config.destination))
 
+            if any(('UseStackBasePointer' in arg for arg in build_args)):
```

Comment:
This check is more or less useless, since GraalVM distributions do not include `mx` (and therefore not this code).
However, I don't think it is crucial to keep the original option alive, since it is not an API option and also probably not widely used yet.

Reply:
Going from your comment, I have reverted the change via 63a9623

/cc @bobmcwhirter